postgres/lib/PostgresService.js

```diff
@@ -240,7 +240,7 @@ GROUP BY k
           text: 'CREATE TEMP TABLE IF NOT EXISTS "$$PARAMETER_BUFFER$$" (PARAM TEXT, NAME TEXT, ID INT) ON COMMIT DROP',
         })
         const proms = []
-        for (const stream of streams) {
+        for (const stream of streams.filter(s => s && s.constructor.name === 'ParameterStream')) {
```

Comment:
@sfdfklskldf @BobdenOs is it expected that there are `undefined` items in the `streams` array? Do you know how this situation can happen?